classes/openrc.bbclass

```diff
@@ -82,6 +83,12 @@ openrc_populate_packages[vardepsexclude] += "OVERRIDES"
 python openrc_populate_packages() {
     import pathlib
 
+    def get_package_var(d, var, pkg):
+        val = (d.getVar('%s:%s' % (var, pkg)) or "").strip()
+        if val == "":
+            val = (d.getVar(var) or "").strip()
+        return val
```

Comment:
PEP8 likes testing the truthyness of nonempty strings.  That with the walrus operator gives something like:
```python
if val := d.getVar(f"{var}:{pkg}").strip():
    return val
if val := d.getVar(var).strip()
    return val
return ""
```

Reply:
Thanks for the tips, I'm not a python specialist...
Changed to:
```
def get_package_var(d, var, pkg):
    if val := (d.getVar(f"{var}:{pkg}") or "").strip():
        return val
    return (d.getVar(f"{var}") or "").strip()
```
As yours first line would crash if d.getVar returns None.